web-client/src/components/training/TrainingInformation.vue

```diff
@@ -37,6 +37,61 @@
       </IconCardSmall>
     </div>
 
+    <!-- Training and validation loss charts -->
+    <div
+      class="flex flex-col md:grid gap-4 md:gap-8"
+      :class="hasValidationData ? 'md:grid-cols-2' : ''"
+    >
+      <!-- Training Accuracy users chart -->
+      <IconCard>
+        <!-- Card header -->
+        <template #title>
+          Training Loss of the Model
+        </template>
+        <template #content>
+          <span class="text-2xl font-medium text-slate-500">
+            {{ (latestEpoch?.training.loss ?? 0).toFixed(2) }}
+          </span>
+          <span class="text-sm font-medium text-slate-500">
+            ' training loss'
```

Comment:
I'm not sure, is it wanted?

```suggestion
            'training loss'
```